### PR TITLE
Ganache: Use determinstic addresses

### PR DESCRIPTION
**Pull request by ggrieco-tob**
This small PR hard-coded a set of private keys to use as account in ganache. This is useful obtain a deterministic results when using etheno. It is based in @montyly's branch.

### CODE REVIEW

File: etheno/genesis.py [hunk 1]
```diff
@@ -106,10 +106,22 @@ def geth_to_parity(genesis):
 
     return ret
 
+DEFAULT_PRIVATE_KEYS = [0xf2f48ee19680706196e2e339e5da3491186e0c4c5030670656b0e0164837257d,
```
Comment (ESultanik) [inline etheno/genesis.py:109]: How were these private keys chosen? Are they just random? If so, why not instead have `make_accounts` accept a seed (defaulting to a hard-coded value) and then generate an arbitrary number of accounts from a PRNG seeded with the given seed?

Comment (ESultanik) [reply etheno/genesis.py:109]: Alternatively, you could just set the private keys incrementally from `0` through `num_accounts - 1`?

Comment (ggrieco-tob) [reply etheno/genesis.py:109]: I think these are the default ganache ones.

Comment (ESultanik) [reply etheno/genesis.py:109]: I guess that's fine, then.

**Review by ESultanik — APPROVED** [review]

**Pull request merged**